core/src/main/java/org/jboss/qa/jcontainer/util/ProcessUtils.java

```diff
@@ -113,6 +120,11 @@ public static void kill(String pid) {
 			} catch (Exception e) {
 				log.error(e.getMessage(), e);
 			}
+			try {
+				Thread.sleep(15 * 1000L);
```

Comment:
Sometimes on Windows next test failed with message like "port 8080 is alreagy opened". I will comment it.